### PR TITLE
add get versions docs

### PR DESCRIPTION
**Pull request by erquhart**
Docs for `kots get versions`: https://github.com/replicatedhq/kots/pull/2464

### CODE REVIEW

File: content/kots-cli/get/versions.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+| Flag              | Type   | Description                                                         |
+| :---------------- | ------ | ------------------------------------------------------------------- |
+| `-h, --help`      |        | help for get apps                                                   |
+| `-n, --namespace` | string | the namespace where the admin console is running _(required)_       |
+
+### Example
+
+```bash
+kubectl kots get kots-sentry -n default
```
Comment (jeffreygolden) [inline content/kots-cli/get/versions.md:27]:
```suggestion
kubectl kots get versions kots-sentry -n default
```

**Comment by JustinN-Replicated** [comment]
@erquhart I think we're missing a flag here (-o json)?

**Comment by JustinN-Replicated** [comment]
Going to add this doc PR to the corresponding SC ticket for this feature.

**Review by jeffreygolden — APPROVED** [review]

**Comment by ajp-io** [comment]
@erquhart I see live docs for `kots get versions`. Can we close this PR?

**Comment by ajp-io** [comment]
Going to close because I already see docs for this.

**Pull request closed without merging**